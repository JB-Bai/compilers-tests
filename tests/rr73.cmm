float;
float Q9IcPakg;
int;

