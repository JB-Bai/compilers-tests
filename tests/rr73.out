Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: float
      SEMI
    ExtDefList (2)
      ExtDef (2)
        Specifier (2)
          TYPE: float
        ExtDecList (2)
          VarDec (2)
            ID: Q9IcPakg
        SEMI
      ExtDefList (3)
        ExtDef (3)
          Specifier (3)
            TYPE: int
          SEMI
